configuration.py

```diff
@@ -1,33 +1,38 @@
 """Contains transformer configuration information
 """
+from agpypeline.configuration import Configuration
 
-# The version number of the transformer
-TRANSFORMER_VERSION = '1.0'
 
-# The transformer description
-TRANSFORMER_DESCRIPTION = ''
+class ConfigurationInfo(Configuration):
+    """Configuration information"""
+    # pylint: disable=too-few-public-methods
+    # The version number of the transformer
+    transformer_version = '1.0'
 
-# Short name of the transformer
-TRANSFORMER_NAME = ''
+    # The transformer description
+    transformer_description = ''
 
-# The sensor associated with the transformer
-TRANSFORMER_SENSOR = ''
+    # Short name of the transformer
+    TRANSFORMER_NAME = ''
```

Comment:
Should this be lowercase to correspond with the rest of the parameters?

Reply:
Thanks for the catch